Conceptpower+Spring/src/main/java/edu/asu/conceptpower/core/ReviewRequest.java

```diff
@@ -49,5 +50,13 @@ public String getResolver() {
     public void setResolver(String resolver) {
         this.resolver = resolver;
     }  
+    
+    public String getResolvingComment() {
+        return resolvingComment;
+    }
+    
+    public void setResolvingComment(String resolvingComment) {
+        this.resolvingComment = resolvingComment;
+    }
```

Comment:
Sure. Having this scenario, I should allow the resolver/requester only to comment without any resolve or reopen operation ?

For now, comment and reopen/resolve is tightly coupled.

Reply:
for now, just open/resolve requests with comments. later on, we'll probalby want to allow to just comment.

Reply:
```resolvingComment``` is not a good name for this property since it's not just for resolving comments, but potentially for any discussion in between as well.

Reply:
getters and setters should have the same name as the variables they belong to.

Reply:
?

Reply:
?